omou-frontend/src/reducers/initialState.js

```diff
@@ -342,8 +342,64 @@ export default {
             },
         ],
     },
+    "CalenderData": {
+        "events_in_view": [{
+            id: 1,
+            title: "6th Grade Math Placement Test Prep",
+            start: '2019-07-22T07:00',
+            end: '2019-07-22T08:00',
+            editable: true,
+            course: "math",
```

Comment:
"math" is the subject. the course type is either "C" or "T" if it's "C" that means this is a course (5+ students), if it's "T" this means this is a tutoring session.